homeassistant/components/voice_assistant/manifest.json

```
@@ -0,0 +1,9 @@
+{
+  "domain": "voice_assistant",
+  "name": "Voice Assistant",
+  "codeowners": ["@balloob", "@synesthesiam"],
+  "config_flow": true,
+  "dependencies": ["stt", "conversation", "tts", "cloud"],
+  "documentation": "https://www.home-assistant.io/integrations/voice_assistant",
+  "iot_class": "local_push"
```

I would set this to `internal`
```suggestion
  "iot_class": "internal"
```

---

I get:

```
Integration voice_assistant:
* [ERROR] [MANIFEST] Invalid manifest: value must be one of ['assumed_state', 'calculated', 'cloud_polling', 'cloud_push', 'local_polling', 'local_push'] for dictionary value @ data['iot_class']. Got 'internal'
```

---

Sorry. Instead, do this:
```suggestion
  "quality_scale": "internal"
```